### DISCUSSION TITLE
About simple-client-server-xml

### DISCUSSION THREAD

**Discussion by FatimaRani** [FreeOpcUa/opcua-asyncio#802] [Q&A]
Hi,
i am trying to run this code "simple-client-server-xml" , it is not working with asyncua package but running with opcua package,
I want to run it with async so I have added await func but still no :(, where should I write this await func?

reeopcua_namespace = self.server.get_namespace_index("urn:freeopcua:python:server")
        ## m@ code
        process_time = random.randint(1, 5)
        await asyncio.sleep(process_time)

        hellower = objects.get_child("0:Hellower")
        hellower_say_hello = hellower.get_child("0:SayHello")

**Comment by AndreasHeine** [comment]
after a short review that example does not work at all... maybe it has not been tested after porting to asyncio! @oroulet 

please use some of the other examples like client-example and server-example!

**Comment by danmoser** [comment]
On Feb 25, 2022 the opcua-asyncio team kindly updated the simple-client-server-xml example!!

If you are using pip, don't forget to enable the --upgrade flag to get the latest version (or just clone the repo)
